scss/_pagination.scss

```diff
@@ -154,10 +156,6 @@
   &:first-child .page-link {
```

Comment:
Should we limit the width to `40px or 2.5rem` of `.page-item:not(:first-child):not(:last-child)` ? Big numbers seems weird atm.